src/test/java/seedu/address/logic/commands/AddPatientCommandTest.java

```diff
@@ -29,6 +32,18 @@
 import seedu.address.testutil.PatientBuilder;
 public class AddPatientCommandTest {
 
+    @Test
+    public void hashcode() {
+        AddPatientCommand command = new AddPatientCommand(ALICE);
+
+        // same value -> returns same hashcode
+        assertEquals(command.hashCode(), Objects.hash(ALICE));
+
+        // different value -> returns different hashcode
+        assertNotEquals(command.hashCode(), Objects.hash(BOB));
```

Comment:
remove this test as this is not expected behaviour. java documentation states: `If two objects are equal according to the equals(Object) method, then calling the hashCode method on each of the two objects must produce the same integer result.`